tests/cxx/default_tpl_arg-d2.h

```diff
@@ -25,3 +25,13 @@ template <typename T = IndirectClass>
 void FnWithProvidedDefaultTplArgAndDefaultCallArg(T* = nullptr) {
   T t;
 }
+
+template <typename T = IndirectClass>
+void FnWithDefArgs1(T = T{}) {
+  T t;
+}
+
+template <typename T = IndirectClass>
+void FnWithDefArgs2(T = {}) {
```

Comment:
Maybe, they should be renamed, if still needed.

Reply:
Aha, you mean they are redundant now because the previous PR already has all combinations of default template args x default function args? Do whatever you feel is best, I can't really visualize the alternatives.

Reply:
Nice :+1: